docs/api/util.rst

```diff
@@ -6,12 +6,6 @@ Utilities
 URI Functions
 -------------
 
-.. NOTE(kgriffs): Due to the way we have been hoisting uri into falcon,
-.. sphinx can't import just falcon.uri (root cause TBD). Whether or not
-.. the way we are hoisting things is legit (hint: probably not, this
-.. is something to address in Falcon 2.0), sphinx can't handle it
-.. by default so we have a custom extension to fix things up.
-
 .. automodule:: falcon.uri
```

Comment:
This was not working before this patch (the section in the docs was blank due to "cannot import falcon.uri").